sdk/textanalytics/azure-ai-textanalytics/pom.xml

```diff
@@ -36,12 +36,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.3.0-beta.2</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
should this using the `unreleased_core` rather than `current`?

Reply:
Yup. change to 
```
<dependency>
      <groupId>com.azure</groupId>
      <artifactId>azure-core</artifactId>
      <version>1.3.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
</dependency>
<dependency>
      <groupId>com.azure</groupId>
      <artifactId>azure-core-http-netty</artifactId>
      <version>1.4.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core-http-netty;dependency} -->
</dependency>
```